README.md

```diff
@@ -129,43 +142,61 @@ You can see an example of profiling in action once the system is running by usin
+* Receives metrics data, via scrape configs, emitted by:
+  * The microservice application.
+  * The Mimir service for operational monitoring.
+  * The Loki service for operational monitoring.
+  * The Tempo service for operatational monitoring.
+  * The Agent itself, for operational monitoring.
+  * The installed Node Exporter service.
+* Receives trace data, via trace configs, emitted by the microservice application.
+* Generates automatic logging lines based on the trace data received.
+* Sends metric, log and trace data onwards to the Mimir, Loki and Tempo services, respectively.
 
-Grafana Agent has remote write capabilities that allows it to send metrics, logs and trace data to backend stores (such as Mimir, Loki and Tempo). More information on Grafana Agent can be found [here](https://grafana.com/docs/agent/latest/).
+Note that as Grafana Agent scrapes metrics for every service defined in the [`docker-compose.yml`](docker-compose.yml) that a significant number of metric [active series](https://grafana.com/docs/grafana-cloud/billing-and-usage/active-series-and-dpm/) are produced (approximately 11,000 at time of writing).
 
-Its main role in this environment is to receive trace spans from the microservice application and process them to extract log information before storing them in the final backend stores.
+The full configuration for Grafana Agent can be found [here](agent/config.yaml).
 
-It also defines several metrics scrape jobs, including:
-* Retrieving metrics from the Prometheus instance itself.
-* Metrics from the microservices application.
-* Metrics from the installed Node Exporter instance.
+### Metrics Generation
 
-All of these metrics are remotely written to the Mimir instance.
+It should be noted that since [v1.4.0](https://github.com/grafana/tempo/blob/main/CHANGELOG.md#v140--2022-04-28), Tempo has included the ability to generate [RED (Rate, Error, Duration)](https://grafana.com/blog/2018/08/02/the-red-method-how-to-instrument-your-services/) [span](https://grafana.com/docs/tempo/latest/metrics-generator/span_metrics/) and [service graph](https://grafana.com/docs/tempo/latest/metrics-generator/service_graphs/) metrics.
 
-Note that the configuration now includes a commented section where metrics used to be generated from incoming trace spans; this is now handled directly in Tempo via server-side metrics generation, although the original configuration block has been left commented out in the Agent config.
+As such, the Grafana Agent configuration now includes a commented section where those metrics used to be generated; this is now handled directly in Tempo via server-side metrics generation.
 
-The configuration file for it can be found in `agent/config.yaml`.
+Whilst this is convenient for many users, there are situations in which it may be preferable to generate metrics locally via Grafana Agent rather than Tempo server-side. These include environments where tail-based sampling may be utilised to discard certain traces.
```

Comment:
```suggestion
Whilst this is convenient for many users, you may prefer to generate metrics locally via Grafana Agent rather than Tempo server-side. These include environments where tail-based sampling may be utilized to discard certain traces.
```